packages/graphics/src/GraphicsGeometry.ts

```diff
@@ -808,7 +809,7 @@ export class GraphicsGeometry extends BatchGeometry
         offset = 0): void
     {
         // TODO use the premultiply bits Ivan added
-        const rgb = (color >> 16) + (color & 0xff00) + ((color & 0xff) << 16);
+        const rgb = Color.shared.setValue(color).toLittleEndianNumber();
 
         const rgba = utils.premultiplyTint(rgb, alpha);
```

Comment:
one for the laterbase - `utils.premultiplyTint` <- this could be  Color function also!